config/localmode/manager_config_patch.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - image: ghcr.io/sovereigncloudstack/cspo-staging:dev
+        name: manager
+        args:
+        - --leader-elect=true
+        - --release-dir=/tmp
```

```suggestion
        - --release-dir=/tmp/downloads
```

We mount the cluster-stacks at `/tmp/downloads` 
Ref: https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/blob/45aa6e45b56bf6d3674f541ea689f0ea73e02c54/Tiltfile#L72

---

or we can omit this as well, this is the default path so we don't need to specify this flag. 
https://github.com/SovereignCloudStack/cluster-stack-operator/blob/2096045cebc4d086f2fcc1d0aab0fb26d9dba6d3/cmd/main.go#L87

---

Good point, I think we can omit this as it's default value is set to `/tmp/downloads/`.